### DISCUSSION TITLE
Show related PR in release notes

### DISCUSSION THREAD

**Discussion by MarvinJWendt** [toeverything/AFFiNE#699] [Ideas]
Hi! Foremost, thank you all for this great project! This might be exactly what I was looking for, for years now.

A thing I noticed is that you don't reference PRs in your GitHub release notes. I think this would be a cool addition, as it would make it simpler to get a look of the new features, the surrounding discussion, and the implementation. 

Something like `Added xyz feature by @someone in #1234`

I would love to explore the new features of future releases, and for that I usually take a look at the PR that implemented it, so I can quickly see how the feature works—and probably the discussion in the comments also shows how to use it.

Let me know what you think :)

**Comment by QiShaoXuan** [comment]
Very good proposal, we will adapt it in the near feature maybe just one month later, since we are still in the midst of a radical update recently

**Comment by lawvs** [comment]
Actually, GitHub can automatically generate release notes with contributors. The blocksuite should be a good example.

<img width="1029" alt="Screenshot 2023-01-09 at 10 37 53 PM" src="https://user-images.githubusercontent.com/18554747/211333640-cde7e35a-6776-48ab-bbf7-38c2a8b12c5a.png">

https://github.com/toeverything/blocksuite/releases/tag/%40blocksuite%2Feditor%400.3.0

**Reply by tzhangchi** [reply]
> Actually, GitHub can automatically generate release notes with contributors. The blocksuite should be a good example.
> 
> <img alt="Screenshot 2023-01-09 at 10 37 53 PM" width="1029" src="https://user-images.githubusercontent.com/18554747/211333640-cde7e35a-6776-48ab-bbf7-38c2a8b12c5a.png">
> 
> https://github.com/toeverything/blocksuite/releases/tag/%40blocksuite%2Feditor%400.3.0

Really important, and I love the feature
